Python/optimizer_analysis.c

```diff
@@ -191,6 +187,13 @@ incorrect_keys(PyObject *obj, uint32_t version)
 #define sym_new_truthiness _Py_uop_sym_new_truthiness
 
 #define JUMP_TO_LABEL(label) goto label;
+#define CHECK_STACK_BOUNDS() do { \
```

Comment:
```suggestion
#define CHECK_STACK_BOUNDS() \
do { \
```
Keep the `do` aligned. It's what we do elsewhere.